packages/nouns-contracts/contracts/governance/NounsDAOV3Proposals.sol

```diff
@@ -540,8 +540,9 @@ library NounsDAOV3Proposals {
         NounsDAOStorageV3.Proposal storage proposal = ds._proposals[proposalId];
 
         proposal.vetoed = true;
+        INounsDAOExecutor timelock = getProposalTimelock(ds, proposal);
```

Comment:
not critical, but a nice practice to add unit tests in our fix PRs, wdyt?

Reply:
@eladmallel generally yes, but in this case we don't have a good testing setup that I can easily duplicate

Reply:
@eladmallel do you think it's ok to merge?